cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -151,27 +151,27 @@ <h3 class="o-post-preview_title">
                 </div>
             {% endif %}
             <div class="post_meta">
-                {% for author in post.get_authors() %}
+                {% for author in post.authors.all() %}
```

Comment:
@kurtrwall doesn't this change break the alphabetization @richaagarwal implementation in #2342? I see that you need both `author.slug` and `author.name` and the `get_authors()` only returns the names. Could that be refactored to return the authors sorted instead of just the names?